src/qibojit/backends/cpu.py

```diff
@@ -1,10 +1,13 @@
 import numpy as np
+from numba import njit
+from qibo.backends import _clifford_operations
 from qibo.backends.numpy import NumpyBackend
 from qibo.config import log
 from qibo.gates.abstract import ParametrizedGate
 from qibo.gates.channels import ReadoutErrorChannel
 from qibo.gates.special import FusedGate
 
+from qibojit.backends import clifford_operations_cpu
```

Comment:
This is unused as well, but I wonder if there is a reason to have it here anyhow.